### DISCUSSION TITLE
Change card size using conditional UI

### DISCUSSION THREAD

**Discussion by DevrayePatil** [Budibase/budibase#7189] [Help]
I want to change card size using conditional UI
Is there any option to change card size using conditional UI

**Comment by nick-walt** [comment]
Do you want to know if a Card can change:
 - depending on the data and context in which the Card is used, or
 - because a user wants to expand the Card using an accordion to drop the lower edge down and expose more in the Card UI?

**Reply by DevrayePatil** [reply]
I want to change only card size using conditional UI

**Comment by melohagan** [comment]
Hey @DevrayePatil 

There is no way to do this currently, however I have raised a feature request to allow bindings in the width/height settings. You could add your condition in there and it would be more flexible in general.
